app/(authenticated)/dashboard/page.tsx

```diff
@@ -1,11 +1,21 @@
 import Leaderboard from '@/components/leaderboard/leaderboard';
 import GraphList from '@/components/ui/GraphList/GraphList';
+import LatestLeadsList from '@/components/ui/LatestLeadsList/LatestLeadsList';
 
 const DashboardPage = () => {
   return (
     <>
```

Comment:
No need to keep the fragment because we are returning only one element inside the fragment.

Reply:
Done